WLogger/Application/AppDelegate.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  AppDelegate.swift
+//  WLogger
+//
+//  Created by Pablo Giorgi on 1/3/16.
+//  Copyright © 2016 Wolox. All rights reserved.
+//
+
+import UIKit
+
+@UIApplicationMain
+class AppDelegate: UIResponder, UIApplicationDelegate {
+
+    var window: UIWindow?
+
+
+    func application(application: UIApplication, didFinishLaunchingWithOptions launchOptions: [NSObject: AnyObject]?) -> Bool {
+        window = UIWindow(frame: UIScreen.mainScreen().bounds)
+        
+        window!.rootViewController = getRootViewController()
```

Comment:
Although here is pretty obvious that is going to work try to make the habit of avoiding `!`